lib/myFTClient.js

```diff
@@ -1025,6 +1025,28 @@ function removeUsersFromLicence (licenceUUID, userUUIDs, options) {
 	return _addRemoveUsers('DELETE', myftConst.licenceNodeName, licenceUUID, userUUIDs, undefined, options, true);
 }
 
+/**
+ * v3/ Remove users from a licence v3/kat/user/remove
+ * Special user case were want to remove ALL followed_by_kat and group membership of as user.
+ * @param {String} licenceUUID - uuid of the licence
+ * @param {String} userUUID - uuid of the user to remove
+ * @param {Object} [options] - additional options {supressEvents: true|false, waitForPurge: true|false }
+ *        default behaviour not to supress event generation and wait for a cache purge
+ * @return {Promise} response -
+**/
+function removeKatUsersFromLicence (licenceUUID, userUUID) {
+
+	const operation = 'myFTClient.removeKatUsersFromLicence';
+	log.debug({operation, licenceUUID, userUUID});
+	const theUrl = `${config.MYFT_API_URL}/kat/user/remove`;
+	const data = {
+		userId: userUUID,
+		licenceId: licenceUUID,
+	};
+
+	return _doRelationshipRequest ('DELETE', theUrl, data);
```

Comment:
space between the function name and the parenthesis (although TIL that that does still work 🙀 )